test/adbc_database_test.exs

```diff
@@ -33,4 +33,53 @@ defmodule Adbc.DatabaseTest do
       assert Exception.message(error) == "[SQLite] Unknown database option who_knows=123"
     end
   end
+
+  describe "load custom driver" do
+    @describetag :duckdb
+
+    setup do
+      {:ok, zip_data} =
+        Adbc.Helper.download(
+          "https://github.com/duckdb/duckdb/releases/download/v0.8.1/libduckdb-linux-amd64.zip",
+          false
+        )
+
+      cache_path = Path.join(File.cwd!(), "libduckdb-linux-amd64.zip")
+      File.write!(cache_path, zip_data)
+
+      adbc_so_priv_dir =
+        case :os.type() do
+          {:win32, _} -> Application.app_dir(:adbc, "priv/bin")
+          _ -> Application.app_dir(:adbc, "priv/lib")
```

Comment:
Let's not write to `priv`. Later we can make a release and accidentally ship the stuff in priv. A "tmp" directory would be better. :)